html/home-page.html

```diff
@@ -0,0 +1,167 @@
+                <span class="bar"></span>
+            </div>
+        </div>
+    </header>
+    <main>
+        
+        <section class="container-fluid head">
+            <div class="container headline">
+            <h1 class="greetings">Hello! World</h1>
+        <h1 class="title"> Let's built  from <br>here</h1>
+        <p class="narates"> GitHub Universe 2022 is in the books but you’re still able to view sessions from all our content tracks. If you registered for the event you can login below, or sign-up as a first time viewer for our session catalog.</p>
+        <h2 class="date">2022.11.09(WED)~10(THU)</h2>
+        <h3 class="venu">Virtual Schedule @watch.githubuniverse.com</h3>
+            </div>
+            <img src="../images/Hero_Gradient.webp" alt="">
+        </section>
+
+        <article class="container-fluid pg ">
+        <div class="main">
+        <h3 class="main-title">Main Program</h3>
+        
+        <span class="line"></span>
+        
+        <div class="container Program-content">
+                <ul>
+                    <li>
+                        <img src="../images/asterisk_0.svg" alt="cloud pic">
+                        <a href="#">Software</a>
+                        <p>Learn how the best companies, developers, and start-ups are using software in the cloud to accelerate productivity form their business. </p>
+                    </li>
+                    <li>
+                        <img src="../images/asterisk_1.svg" alt="security pic">
+                        <a href="#">Security</a>
+                        <p>Hone your security skills by exploring the GitHub tools, techniques, and best practices for securing your software development lifecycle. 
+                        </p>
+                    </li>
+                    <li>
+                        <img src="../images/asterisk_2.svg" alt="AI pic">
+                        <a href="#">Artificial</a>
+                        <p>See how GitHub, businesses, and the machine learning and AI community build exciting solutions to address current software challenges. </p>
+                    </li>
+                    <li>
+                        <img src="../images/asterisk_2.svg" alt="AI pic">
+                        <a href="#">Artificial</a>
+                        <p>See how GitHub, businesses, and the machine learning and AI community build exciting solutions to address current software challenges. </p>
+                    </li>
+                    <li>
+                        <img src="../images/asterisk_2.svg" alt="AI pic">
+                        <a href="#">Artificial</a>
+                        <p>See how GitHub, businesses, and the machine learning and AI community build exciting solutions to address current software challenges. </p>
+                    </li>
+                </ul>
+                
+         </div>
+         <a href="#" class="join" >SEE THE WHOLE PROGRAM</a>
+        </div>
+        </article>
```

Comment:
- What the previous reviewer meant was that you should wrap each of your `img`, `a` and `p` tags in an `article` tag.
` < li >

           <article>
                                                                                      
              <img/>

              <a></a>

              <p></p>

           </article>
       </li>`
- Please use the `section` tag to wrap the whole of this section instead of an `article` tag as you have done. PLease fix this 🙏 . It would help improve the semantics of your site 💯

Reply:
i have added the article tag inside the li